install/wastebin-install.sh

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env bash
+
+# Copyright (c) 2021-2024 tteck
+# Author: tteck
+# Co-Author: MickLesk (Canbiz)
+# License: MIT
+# https://github.com/tteck/Proxmox/raw/main/LICENSE
+# Source: https://github.com/matze/wastebin
+
+source /dev/stdin <<< "$FUNCTIONS_FILE_PATH"
+color
+verb_ip6
+catch_errors
+setting_up_container
+network_check
+update_os
+
+msg_info "Installing Dependencies (Patience)"
+$STD apt-get install -y --no-install-recommends \
+  build-essential \
+  curl \
+  sudo \
+  git \
+  make \
+  mc
+msg_ok "Installed Dependencies"
+
+msg_info "Installing Rust (Patience)" 
+bash <(curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs) -y
```

Comment:
This needs a pre `$STD` or post `> /dev/null 2>&1`

Reply:
done